### PR TITLE
doc/cephfs: improve cache-configuration.rst

### PR DESCRIPTION
**Pull request by zdover23**
Improve the text in the section about dealing with cache-pressure alerts that was added in https://github.com/ceph/ceph/pull/59077. The changes in this commit were suggested by Anthony D'Atri.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

**Review by anthonyeleven — APPROVED** [review]

File: doc/cephfs/cache-configuration.rst [hunk 1]
```diff
@@ -255,10 +255,10 @@ respond to cache pressure" messages no longer appear in the logs.
 Example Scenario
 ~~~~~~~~~~~~~~~~
 
-Here is an example. A client is having 20k caps cached. At some moment the
+Here is an example. A client is having 20,000 caps cached. At some moment the
```
Comment (batrick) [inline doc/cephfs/cache-configuration.rst:258]: Given that different locales use "," or ".", would it not be more understandable to use an SI prefix?

File: doc/cephfs/cache-configuration.rst [hunk 2]
```diff
@@ -237,8 +237,8 @@ all cached caps (the actual recall value is ``num_caps -
 mds_min_caps_per_client(100)``).
 
 Under certain circumstances, many "recall caps" requests can be sent so quickly
-that the "mon warning limit" exceeded, and the "clients failing to respond to
-cache pressure" message can be triggered. If the client does not release the
+that the "mon warning limit" is exceeded, and the "clients failing to respond
+to cache pressure" message can be triggered. If the client does not release the
```
Comment (batrick) [inline doc/cephfs/cache-configuration.rst:241]:
```suggestion
that the health warning is generated: "clients failing to respond
to cache pressure". If the client does not release the
```

File: doc/cephfs/cache-configuration.rst (hunk 1 above)
Comment (zdover23) [reply doc/cephfs/cache-configuration.rst:258]: What do you think about "20K"?

File: doc/cephfs/cache-configuration.rst (hunk 2 above)
Comment (zdover23) [reply doc/cephfs/cache-configuration.rst]: I've committed this change. Thanks, @batrick.

File: doc/cephfs/cache-configuration.rst (hunk 1 above)
Comment (batrick) [reply doc/cephfs/cache-configuration.rst]: The prefixes are generally case-sensitive so I think "k" is still appropriate: https://en.wikipedia.org/wiki/Metric_prefix

**Review by batrick — APPROVED** [review]

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/59214 - Squid backport
https://github.com/ceph/ceph/pull/59215 - Reef backport
https://github.com/ceph/ceph/pull/59216 - Quincy backport

**Pull request merged**